script-preserveData.js

```diff
@@ -0,0 +1,19 @@
+// store data
+const form = document.getElementById('form');
+const data = {
+  name: document.getElementById('name').value,
+  email: document.getElementById('email').value,
+  text: document.getElementById('msg').value,
+};
+
+function storeData() {
+  localStorage.setItem('form', JSON.stringify(data));
+}
+
+form.addEventListener('input', storeData);
+
+const formObj = JSON.parse(localStorage.getItem('form'));
+
+document.getElementById('name').value = formObj.name;
+document.getElementById('email').value = formObj.email;
+document.getElementById('msg').value = formObj.text;
```

Comment:
- [ ] Good job far, :clap: However, Your form data send data using `formspree` but does not save data in the broswer's `Localstorage`, and also, your page does not persist data on the form when `refreshed or reloaded`

Please check your `script-preservedData.js` file and ensure that it is imported correctly in your `index.html` file :slightly_smiling_face: 

Below is a screenshot of your site: :arrow_down: 

![Student Data](https://github.com/binyamolango/Personal-Portfolio/assets/43528594/db3ce028-3422-46d4-9235-9df7c29c945d)